cmd/controller-manager/app/controller_manager.go

```diff
@@ -360,9 +367,55 @@ func StartControllers(s *options.ControllerManagerServer,
 	glog.V(1).Info("Starting shared informers")
 	informerFactory.Start(stop)
 
+	if availableResources[podPresetGVR] {
```

Comment:
I think we may want to tease this into a separate controller, or at least one that you can run without running the entire service catalog controller, but this is sufficient for now.